cmd/zstreamdump/zstreamdump.c

```diff
@@ -55,6 +55,8 @@
 
 uint64_t total_write_size = 0;
 uint64_t total_stream_len = 0;
+uint64_t total_overhead_size = 0;
+uint64_t drr_byte_count[DRR_NUMTYPES] = { 0 };
```

Comment:
nit: I understand why you added these here, and functionally it doesn't really matter, but rather then adding new global variables I'd suggest adding them after `drr_record_count[drr_record_count]` in `main()`.  It wouldn't be bad idea to move `total_write_size` there too.